app/posts/find-teacher-training/2021-08-16-standardised-entry-requirement-descriptions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Standardising entry requirement content
+description: Why we have made the descriptions of entry requirements more consistent
+date: 2021-08-16
+---
+
+Prior to the 2022 to 2023 cycle, providers were asked to describe the academic qualification entry requirements for each course as a single free text field.
```

Comment:
1) Use 'Before' instead of 'Prior to'.

2) Use 'in a single free text field' instead of 'as a single free text field'.